symbol_exporter/ast_package_symbol_extractor.py

```diff
@@ -101,9 +101,12 @@ def _normalize_and_sort_symbols(self):
                 if get_symbol_type(v) not in {SymbolType.PACKAGE, SymbolType.MODULE}:
                     namespace = new_symbol.rpartition(".")[0]
                     namespaces[namespace].append(new_symbol)
-        for rel_import in topological_sorter.static_order():
-            if rel_import in relative_star_imports_volume:
-                tmp_sorted[rel_import] = relative_star_imports_volume[rel_import]
+        try:
+            for rel_import in topological_sorter.static_order():
+                if rel_import in relative_star_imports_volume:
+                    tmp_sorted[rel_import] = relative_star_imports_volume[rel_import]
+        except graphlib.CycleError:
```

Comment:
```suggestion
        except CycleError:
```

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.